### PR TITLE
Bot mode setup documentation rework

### PR DESCRIPTION
**Pull request by Gnuxie**
https://github.com/the-draupnir-project/planning/issues/11#issue-2109912167

### CODE REVIEW

**Comment by Gnuxie** [comment]
also the underscores in the filenames are killing me

File: docs/bot/systemd.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+The first thing that you will need to do is create the directory
+`/var/lib/draupnir`, which will be used to store Draupnir's
+persistent data.
+
+You will then want to copy your configuration file for Draupnir to
+`/var/lib/draupnir/config/production.yaml`.
+
+Then you will want to copy the following systemd unit file to
+`where?` and call it `draupnir.service`.
```
Comment (Gnuxie) [inline docs/bot/systemd.md:36]: Where are we copying the service file to? Well if we're storing persistent data in `/var/lib/draupnir` then we should probably just make it a system service file.

File: docs/bot/systemd.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+simple as creating the following systemd unit file.
+
+:::warning[TODO]
+
+Questions:
+
+1. Do we need a specific user to write to the data directory? Does
+docker even work well with that?
+
+:::
```
Comment (Gnuxie) [inline docs/bot/systemd.md:18]: This is unnecessary, docker runs everything as root sure but the directories that the container has access to is constrained to the volumes that are mounted

**Review by FSG-Cat — APPROVED** [review]
Cat is happy with the state of this except that we need to probably add a note about how to avoid triggering whats showed in this image. 

Reproduction steps are Synapse, No rate limit Draupnir and Element Web on same access token in a active tab in Firefox and in this case same system as Synapse and Draupnir. 
image

The exact wording is something i wont create. 

Leaving this as approved as this can be split out to another PR.

**Pull request merged**